src/imitation/scripts/config/train_rl.py

```diff
@@ -6,7 +6,11 @@
 
 train_rl_ex = sacred.Experiment(
     "train_rl",
-    ingredients=[common.common_ingredient, train.train_ingredient, rl.rl_ingredient],
+    ingredients=[
```

Comment:
It didn't, it's a side-effect of having added too many elements when we temporarily created a new ingredient, then flake8 forces adding a comma at the end, then when you remove the elements black does not reformat the file back due to the magic comma. I think this is actually quite undesirable, and I have been annoyed at flake8's expectation of a final comma everywhere triggering a reformat from black, even if the elements enumerated fit in one line (but not the surrounding brackets, e.g. square or round).

Reply:
For example:
```python
func_call(list_of_long, arguments_that_all, fit_in_one_line, but_not_also, the_fn_name)
```
black will reformat this to:
```python
func_call(
     list_of_long, arguments_that_all, fit_in_one_line, but_not_also, the_fn_name
)
```
flake8 will require a trailing comma:
```python
func_call(
     list_of_long, arguments_that_all, fit_in_one_line, but_not_also, the_fn_name,
)
```
but then black breaks the entire line:
```python
func_call(
     list_of_long, 
     arguments_that_all, 
     fit_in_one_line, 
     but_not_also, 
     the_fn_name,
)
```
this can be super unreadable sometimes and make the code too vertically long. It also has the problem that if you remove elements from this call (e.g. optionals):
```python
func_call(
     list_of_long, 
)
```
neither flake8 nor black will complain/reformat this.